spec/suites/map/MapSpec.js

```diff
@@ -458,6 +458,20 @@ describe("Map", function () {
 			map.setMaxBounds(bounds, {animate: false});
 			expect(map.off.called).not.to.be.ok();
 		});
+
+		it("avoid subpixel / floating point related wobble (#8532)", function (done) {
+			map.setView([50.450036, 30.5241361], 13);
+
+			var spy = sinon.spy();
+			map.on('moveend', spy);
```

Comment:
Should be `once` so that it doesn't pollute other tests, right?
```suggestion
			map.once('moveend', spy);
```

Reply:
Ah right.